client/src/components/ratings-reviews/RatingBreakdown.jsx

```diff
@@ -45,7 +45,32 @@ const RatingBreakdown = (props) => {
 
   return (
     <div id="rating-breakdown-container">
-      <span id="average-review">Average Rating: {Math.round(averageReview * 10) / 10} </span>
+      <Row>
+        <Col md={4}>
+          <span>Average Rating: </span>
+        </Col>
+        <Col md={8}>
+          <span className="score">
+            <div className="score-wrap">
+              <span className="stars-active" style={{width: `${averageReview * 20}%`}}>
```

Comment:
Not sure if this is a mistake or not, but this should be the total review % as a whole number. I think it should be averageReview * 10?